src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__img {
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    flex-grow: 1;
+    margin: 0 0 4px;
+    max-width: 166px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__line {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__btn {
+    width: fit-content;
+    margin: 0 auto;
+    text-decoration: none;
+    padding: 12px 70px;
```

Comment:
Not quite correct to add such large left/right paddding for button, because for example, if the text becomes larger, it will be like on the screen. Try to make button on full width of card(use width: 100% for exmpl), and place text in the center(use text-align: center)
![image](https://github.com/mate-academy/layout_product-cards/assets/54936044/a69c8c7c-449d-4a3b-b75c-997ac327a46a)